### PR TITLE
Add dtl/1.19 and dtl/cci.20210404

### PR DESCRIPTION
**Pull request by LHLaurini**
Specify library name and version:  dtl/1.19 and dtl/cci.20210404

This library is a dependency to one of my projects, which I intend to eventually also submit to the CCI.

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/dtl/all/test_package/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.18)
```
Comment (prince-chrismc) [inline recipes/dtl/all/test_package/CMakeLists.txt:1]:
```suggestion
cmake_minimum_required(VERSION 3.1)
```

File: recipes/dtl/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    license = "BSD"
+    no_copy_source = True
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        src_dir = glob.glob(f"{self.name}-*/")[0]
+        os.rename(src_dir, "dtl")
+
+    def package(self):
+        self.copy("dtl/*", dst="include", src="dtl")
```
Comment (prince-chrismc) [inline recipes/dtl/all/conanfile.py:21]: Path separator should be cross platform =)

```suggestion
        self.copy(os.path.join("dtl", "*.hpp"), dst="include", src="dtl")
```

**Review by prince-chrismc — CHANGES_REQUESTED** [review]
Very close! great work, just a few minor tweaks

**Comment by LHLaurini** [comment]
> Very close! great work, just a few minor tweaks

Thank you for the review. Changes accepted. Some of these were caused by a lack of attention, sorry for that 😄  (specially the wrong license name, forgot to look it up).

**Review by prince-chrismc — APPROVED** [review]
Sorry about the bad edit :p

File: recipes/dtl/all/conandata.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+sources:
+  "1.19":
+    url: "https://github.com/cubicdaiya/dtl/archive/v1.19.tar.gz"
+    sha256: "f47b99dd11e5d771ad32a8dc960db4ab2fbe349fb0346fa0795f53c846a99c5d"
+  "cci.20210404":
```
[the remark this answers is not in the document]
Comment (uilianries) [reply recipes/dtl/all/conandata.yml:5]: thank you.

**Review by uilianries — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**